### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by altmas5**
Remove consecutive occurrences of 'the' in 7 files.
`s/the the /the /g`

### CODE REVIEW

File: content/id/docs/reference/access-authn-authz/rbac.md [hunk 1]
```diff
@@ -1185,10 +1185,11 @@ memuat [kebijakan ABAC lama](/docs/reference/access-authn-authz/abac/#policy-fil
 --authorization-mode=...,RBAC,ABAC --authorization-policy-file=mypolicy.json
 ```
 
-To explain that first command line option in detail: if earlier authorizers, such as Node,
-deny a request, then the RBAC authorizer attempts to authorize the API request. If RBAC
-also denies that API request, the ABAC authorizer is then run. This means that any request
-allowed by *either* the RBAC or ABAC policies is allowed.
```
Comment (VineethReddy02) [inline content/id/docs/reference/access-authn-authz/rbac.md]: Can I know why are we changing the english content to other language? 
@altmas5

**Comment by altmas5** [comment]
Sorry, it looks like I made a mess trying to merge durin the rebase. Please reject the changes

**Review by sftim — COMMENTED** [review]
/kind cleanup

I hope this is OK for the non-English localizations; it should be. Normally we [limit pull requests to one language](https://kubernetes.io/docs/contribute/new-content/overview/#languages-per-pr)
/lgtm
/approve

**Pull request merged**